variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/VariantKeyFieldsTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Copyright 2017 EMBL - European Bioinformatics Institute
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.ac.ebi.eva.commons.core.models;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class VariantKeyFieldsTest {
```

Comment:
Decide whether to add more tests based on [VariantVcfFactoryTest](https://github.com/pabarcgar/variation-commons/blob/67ac0d703253521fcd91f6bf735f556ed67a74e2/variation-commons-core/src/test/java/uk/ac/ebi/eva/commons/core/models/factories/VariantVcfFactoryTest.java).